altofocus/style.css

```diff
@@ -3189,10 +3189,16 @@ body {
   clear: both;
   display: block;
   padding: 1.75em 10%;
-  position: absolute;
   text-align: center;
   width: 100%;
   z-index: 1;
+  position: relative;
```

Comment:
This needs to be changed in `altofocus/assets/stylesheets/shared/_infinitescroll.scss`, and then compiled here.